app/include/zmk/hid_indicators_types.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+typedef uint8_t zmk_hid_indicators;
```

Comment:
We usually follow the convention of using a `_t` suffix for types like these:

```suggestion
typedef uint8_t zmk_hid_indicators_t;
```